microsoft/terminal
# Control 'Touch Keyboard and Handwriting Panel Service' warning

## Summary of the Pull Request
Add a setting to turn off the warning  if 'Touch Keyboard and Handwriting Panel Service' is disabled.
 
## References
#8095
https://github.com/microsoft/terminal/issues/7886#issuecomment-729350169

## PR Checklist


## Detailed Description of the Pull Request / Additional comments
The service might not start in some case, and it doesn't affect the input in some computer.
This PR turn off the warning even if the service is disabled.
The setting name is  "inputServiceWarning".

part of settings.json:

    // If enabled, selections are automatically copied to your clipboard.
    "copyOnSelect": false,
    
    // If disabled, don't show warning if 'Touch Keyboard and Handwriting Panel Service' disabled.
    "inputServiceWarning": false,





## Validation Steps Performed
I manually set the service to "Disabled", restarted the Terminal, verified the warning up, then set "inputServiceWarning" to false and restarted the Terminal, and the warning didn't appear.
